components/salesforce_rest_api/common/sobjects/campaign.mjs

```diff
@@ -0,0 +1,92 @@
+export default {
+  ActualCost: {
+    type: "string",
+    label: "ActualCost",
+    description: "Amount of money spent to run the campaign.",
+  },
+  BudgetedCost: {
+    type: "string",
+    label: "BudgetedCost",
+    description: "Amount of money budgeted for the campaign.",
+  },
+  CampaignImageId: {
+    type: "string",
+    label: "CampaignImageId",
+    description: "ID of the campaign image. Available in API version 42.0 and later.",
+  },
+  CampaignMemberRecordTypeId: {
+    type: "string",
+    label: "CampaignMemberRecordTypeId",
```

Comment:
You can separete de words in the label.

You can do this for the other labels too.

```suggestion
    label: "Campaign Member Record Type ID",
```